src/interactions/hoverInteraction.ts

```diff
@@ -34,6 +34,7 @@ export module Interaction {
   export class Hover extends Interaction.AbstractInteraction {
     public _componentToListenTo: Hoverable;
     private _dispatcher: Dispatcher.Mouse;
+    private _touchDispatcher: Dispatcher.Touch;
```

Comment:
Are we sure we want a TouchDispatcher on a HoverInteraction?  I was first imagining the idea of having a TouchInteraction that would handle the TouchDispatcher ideas.